### PR TITLE
feat: Auth

### PR DESCRIPTION
**Pull request by PeeyushPrashant**
- Login Page
- SignUp Page

NOTE:- The commented code will be used later on , so kindly bear with it

Screenshot (137)
Screenshot (148)

### CODE REVIEW

File: src/components/NavBar/Navbar.jsx [hunk 1]
```diff
@@ -37,7 +47,7 @@ export const NavBar=()=>{
           <div className="saved-item flex-row"
         //   onClick={sideBarHandler}
```
Comment (anuj-kumary) [inline src/components/NavBar/Navbar.jsx:48]: Remove unwanted comments

File: src/contexts/auth-context.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            token: response.data.encodedToken,
+            user: response.data.createdUser,
+          })
+        );
+        setToken(response.data.encodedToken);
+        setUser(response.data.createdUser);
+        navigate("/");
+      }
+    } catch (error) {
+      console.log(error);
```
Comment (anuj-kumary) [inline src/contexts/auth-context.js]: Just a suggestion show error in the console using ` console.error()`

File: src/pages/Auth/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+       <NavBar/>
+        
+        <div className="auth-container flex-row">
+          <div className="auth-card flex-col">
+            <h1 className="auth-heading">Login</h1>
+            <p>Please enter your valid email and password.</p>
+            <div className="auth-input flex-row">
+              <label for="" className="input-label"><strong>Email</strong></label>
+              <input
+                type="text"
```
Comment (anuj-kumary) [inline src/pages/Auth/Login.jsx]: For email input, you can use the type="email" it gives you email validation also

File: src/App.js [hunk 4]
```diff
@@ -1,12 +1,14 @@
 import "./App.css";
-import { Home } from "../src/pages/index";
+import { Home, Login, SignUp } from "../src/pages/index";
```
Comment (anuj-kumary) [inline src/App.js]: In barrel import no need to mention index

**Review by anuj-kumary — APPROVED** [review]
The code is clean and great folder structure.
Just added some suggestions.

**Review by ramalingaa — APPROVED** [review]
Code looks clean, logic is working. PR is good to merge

**Pull request merged**